gensim/test/test_drmm_tks.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2010 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Automated tests for checking transformation algorithms (the models package).
+"""
+
+import unittest
+import gensim.downloader as api
+from gensim.test.utils import datapath, get_tmpfile
+from gensim.models.experimental import DRMM_TKS
+
+
+class TestDrmmTksModel(unittest.TestCase):
+
+    def testLoadModel(self):
+        model = DRMM_TKS.load(datapath('drmm_tks'))
```

Comment:
Better to have save/load in one test:
- train model
- save it
- load it
- "sanity check" (models produce same output on same input)

If save/load called without exceptions, this means nothing, unfortunately, need to deeper check.